<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Employee</class>
 <widget class="QDialog" name="Employee">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="title">
      <string>Employee Login</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>80</y>
        <width>211</width>
        <height>131</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>21</y>
         <width>191</width>
         <height>70</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="styleSheet">
             <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;</string>
            </property>
            <property name="text">
             <string>Username:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_Username"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="styleSheet">
             <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;</string>
            </property>
            <property name="text">
             <string>Password:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_Password">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="EmployeeLogin">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>100</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_Picture">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>111</width>
        <height>111</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>230</y>
        <width>161</width>
        <height>25</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="text">
          <string>New user?</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="EmployeeSigin">
         <property name="styleSheet">
          <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 0, 0);</string>
         </property>
         <property name="text">
          <string>Register</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
